1_space_odyssey/js/_adventures-9000/app.js

```diff
@@ -152,8 +156,21 @@ var responses = [
   "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
 ];
 
+
 // write your code here, use the console log beneath
-// console.log("\u{1F6F8} Swoosh -> -> Ship is steered to safety!");
+
+var nrOfResponses = responses.Length;
+
+for (var Nr = 0; Nr <= nrOfResponses; Nr++) {
```

Comment:
in javaScript variable names are camel case starting with small letter / lowercase letter
use: `i, j, or k` for the counter variable, not Nr